spec/account/account-manager.spec.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+describe("AccountManager", () => {
+    var accountManager, otDateMock;
+
+    beforeEach(() => {
+        var AccountManager, accountServiceFake, config, hotpFake, promiseMock, randomMock;
+
+        AccountManager = require("../../lib/account/account-manager");
+        otDateMock = require("../mock/ot-date-mock");
+        promiseMock = require("../mock/promise-mock");
+        randomMock = require("../mock/random-mock");
+        accountServiceFake = jasmine.createSpyObj("accountServiceFake", [
+            "completeAsync",
+            "getAsync",
+            "initiateAsync"
+        ]);
+        accountServiceFake.completeAsync = jasmine.createSpy("complete").andCallFake(() => {
```

Comment:
This was a complication from when I was editing on the wrong branch. It's corrected now.